.editorconfig

```diff
@@ -9,6 +9,8 @@
 
 [*.kt]
 indent_size = 2
+ij_kotlin_allow_trailing_comma_on_call_site = true
+ij_kotlin_allow_trailing_comma = true
```

Comment:
what is the significance of this config?

Reply:
> 

@kth13 These were automatically brought in from the updated dps-gradle-spring-boot plugin (4.8.4) and enforces usage of trailing commas